charts/gatekeeper/templates/_helpers.tpl

```diff
@@ -36,10 +36,44 @@ Adds additional pod labels to the common ones
 */}}
 {{- define "gatekeeper.podLabels" -}}
 {{- if .Values.podLabels }}
-{{- toYaml .Values.podLabels | nindent 8 }}
+{{- toYaml .Values.podLabels }}
```

Comment:
@sozercan  this change seems to break the chart rendering when `podLabels` is set in values. 

`
Error: YAML parse error on gatekeeper/templates/gatekeeper-audit-deployment.yaml: error converting YAML to JSON: yaml: line 32: mapping values are not allowed in this context
helm.go:84: [debug] error converting YAML to JSON: yaml: line 32: mapping values are not allowed in this context
`
rendered snippet with invalid yaml due to indentation issue:
```
.....
 template:
    metadata:
      annotations:
        prometheus.io/port: "8888"
        prometheus.io/scrape: "true"
      labels:policyengine: opa         <======== podLabels
        app: 'gatekeeper'
````

Reply:
Ah, fixed in https://github.com/open-policy-agent/gatekeeper/pull/3153.  Waiting on next 3.14.x release.